src/api/json/catalog.json

```diff
@@ -5122,6 +5122,12 @@
       "description": "Dynamic Bash aliases",
       "fileMatch": [".aliases"],
       "url": "https://json.schemastore.org/aliases.json"
+    },
+    {
+      "name": "config.yml",
+      "description": "Lazy Git settings",
+      "fileMatch": ["**/.config/lazygit/config.yml", "**/.config/lazygit/config.yaml"],
```

Comment:
Those file locations are only valid for Linux. On Mac and Windows, the paths are slightly different, see [docs/Config.md](https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#user-config). It seems that `"**/lazygit/config.yml"` is a pattern that should work for all platforms.

Also, I don't think we need `.yaml` in the pattern, as it is not supported by lazygit.